tests/postgres_tests/test_aggregates.py

```diff
@@ -41,6 +41,30 @@ def setUpTestData(cls):
             ),
         ])
 
+    def test_empty_result_set(self):
+        AggregateTestModel.objects.all().delete()
+        tests = [
+            (ArrayAgg('char_field'), []),
+            (BitAnd('integer_field'), None),
+            (BitOr('integer_field'), None),
+            (BoolAnd('boolean_field'), None),
+            (BoolOr('boolean_field'), None),
+            (JSONBAgg('integer_field'), []),
+            (StringAgg('char_field', delimiter=';'), ''),
+        ]
+        for aggregation, expected_result in tests:
+            with self.subTest(aggregation=aggregation):
+                with self.assertNumQueries(0):
+                    values = AggregateTestModel.objects.none().aggregate(
+                        aggregation=aggregation,
+                    )
+                    self.assertEqual(values, {'aggregation': expected_result})
+                with self.assertNumQueries(1):
```

Comment:
Compared to this other query:

```suggestion
                # Check empty result when query must be executed.
                with self.assertNumQueries(1):
```